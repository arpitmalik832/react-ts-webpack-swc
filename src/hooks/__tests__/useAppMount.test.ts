import { cleanup, renderHook } from '@testing-library/react';
import '@testing-library/jest-dom';

import hook from '../useAppMount';

jest.mock('@arpitmalik832/react-js-rollup-monorepo-library', () => ({
  __esModule: true,
  useTheme: jest.fn(),
  useInitAxios: jest.fn(),
}));

describe('useAppMount unit tests', () => {
  afterEach(cleanup);

  it('to match snapshot', () => {
    const renderedHook = renderHook(hook);

    expect(renderedHook).toMatchSnapshot();
  });
});
