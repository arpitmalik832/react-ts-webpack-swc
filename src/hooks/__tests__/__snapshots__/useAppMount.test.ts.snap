// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useAppMount unit tests to match snapshot 1`] = `
{
  "rerender": [Function],
  "result": {
    "current": undefined,
  },
  "unmount": [Function],
}
`;
